server/use_cases/add_workshifts.py

```diff
@@ -0,0 +1,14 @@
+"""
+This module contains the use case for adding work shifts.
+"""
+def workshift_add_use_case(repo, work_shift):
+    """
+    The function adds a workshift into the in-memory database.
```

Comment:
Please reword, to avoid references to in-memory database (this could be any data repository, we are just using in-memory repo for initial development)